css/styles.css

```diff
@@ -111,21 +110,41 @@ header {
   align-self: flex-end;
 }
 
+.navbar {
+  list-style-type: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: var(--my-text-color);
+  padding: 1% 5%;
+}
+
```

Comment:
- [ ] Please make sure that mobile design still looks like expected after adding the desktop version as we are aiming for a responsive website so adding one shouldn't effect the other design

Your design:

<img width="381" alt="Screenshot 2023-03-31 at 9 24 07 AM" src="https://user-images.githubusercontent.com/44798044/229029842-ebc70801-f6cc-4f5b-a610-73eb1e023229.png">


Mobile version Figma design:


<img width="134" alt="Screenshot 2023-03-31 at 9 24 31 AM" src="https://user-images.githubusercontent.com/44798044/229029454-1e860cf5-b413-4fc0-bec9-a2b130fbf022.png">